src/convertToObject.js

```diff
@@ -1,18 +1,31 @@
 'use strict';
+/* eslint-disable */
 
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const callback = (prev, elem ) => ({
+      ...prev, [elem[0].trim()]: elem[1].trim(),
+  });
```

Comment:
Move this callback outside of the convertToObject function, to avoid creating it on each call.